src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkDiagnosticListener.cs

```diff
@@ -202,7 +214,7 @@ public override void OnCustom(string name, Activity activity, object payload)
                         {
                             if (activity.IsAllDataRequested)
                             {
-                                activity.SetStatus(Status.Ok);
+                                activity.SetStatus(Status.Unset);
```

Comment:
Unset is the default, so no need to setting to Unset I think.

Reply:
I have tried to remove this line, and the unit tests become red

Reply:
this must be a bug and should be investigated... Its expected to be Unset by default. If its not, we need to check if somewhere in this instrumentation, it is set otherwise.